src/PartielsPy/export_configs/_export_config.py

```diff
@@ -0,0 +1,34 @@
+"""An abstract class for export configuration"""
+
+
+class ExportConfig:
+    """Export configuration class
+
+    This class is used to configure the export of the document.
+    It is used to set the adapt option.
+    This class is inherited by the other ExportConfig classes:
+    - ExportConfigImage
+    """
+
+    def __init__(self):
+        self.__adapt = False
+
+    @property
+    def adapt(self) -> bool:
+        """Whether the adapt option is enabled.
+
+        Set to True to enable.
```

Comment:
The line is too obvious. And you can copy or rephrase the help from the CLI so that the users won't have to run the CLI to know what it does. And you can also specify in the class documentation that the default value is false (so this is more consistent with the other part of the code).